spec/test.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../lib/board'
+require_relative '../bin/main'
+require_relative '../bin/Rspec.file'
```

Comment:
I checked `bin/` directory, but I couldn't find `Rspec.file` in there. if it is a mistake, kindly remove it.